projects/rocksdb/src/main/kotlin/Rocks.kt

```diff
@@ -0,0 +1,28 @@
+import org.rocksdb.InfoLogLevel
+import org.rocksdb.Options
+import org.rocksdb.RocksDB
+
+class Rocks private constructor(private val db: RocksDB, private val afterClose: () -> Unit): AutoCloseable {
+    companion object {
+        init {
+            RocksDB.loadLibrary()
+        }
+
+        fun open(name: String): Rocks {
+            val opts = Options().apply {
+                setCreateIfMissing(true)
+                setInfoLogLevel(InfoLogLevel.INFO_LEVEL)
+            }
+            return Rocks(RocksDB.open(opts, name), opts::close)
```

Comment:
From what I understand the options used here will be copied internally and are free to be "reused."

> **Q: Can I reuse DBOptions or ColumnFamilyOptions to open multiple DBs or column families?**
>
> A: Yes. Internally, RocksDB always makes a copy to those options, so you can freely change them and reuse these objects.
> — [RocksDB FAQ](https://github.com/facebook/rocksdb/wiki/RocksDB-FAQ)

With that, we should be able to avoid passing in `opts::close` and instead close the `Options` from now.